src/api/ethereum.ts

```diff
@@ -215,16 +214,6 @@ export const startFlow = async (
 							maxPriorityFeePerGas,
 						},
 					}),
-					await framework.idaV1.approveSubscription({
-						superToken: SUSHIxAddress,
-						indexId: '2',
-						publisher: exchangeAddress,
-						userData,
-						overrides: {
-							maxFeePerGas,
-							maxPriorityFeePerGas,
-						},
-					}),
```

Comment:
Please leave this since this approval sequence is needed for REX LP, I will come through soon to work on this since I'm relaunching REX LP